src/test/ui/or-patterns/basic-switchint.rs

```diff
@@ -0,0 +1,57 @@
+// Test basic or-patterns when the target pattern type will be lowered to
+// a `Switch`. This will happen when the target type is an integer.
+
+// run-pass
+
+#![feature(or_patterns)]
+
+#[derive(Debug, PartialEq)]
+enum MatchArm {
+    Arm(usize),
+    Wild,
+}
+
+#[derive(Debug)]
+enum Foo {
+    One(usize),
+    Two(usize, usize),
+}
+
+fn test_foo(x: Foo) -> MatchArm {
+    match x {
+        // normal pattern.
+        Foo::One(0) | Foo::One(1) | Foo::One(2) => MatchArm::Arm(0),
+        // most simple or-pattern.
+        Foo::One(42 | 255) => MatchArm::Arm(1),
+        // multiple or-patterns for one structure.
+        Foo::Two(42 | 255, 1024 | 2048) => MatchArm::Arm(2),
+        // mix of pattern types in one or-pattern (range).
+        //
+        // FIXME(dlrobertson | Nadrieril): Fix or-pattern completeness and
```

Comment:
cc @Nadrieril is this still unresolved?

Reply:
This comment is a leftover from https://github.com/rust-lang/rust/pull/63688, which dated from before I implemented exhaustiveness checking for or-patterns. So it's resolved :)

Reply:
Ah, fantastic! ❤️